From 0800c75ad5f57155c3acfb8f3193fddc975921f1 Mon Sep 17 00:00:00 2001
From: Gao Pan <b54642@freescale.com>
Date: Fri, 7 Aug 2015 14:41:00 +0800
Subject: [PATCH 0206/5242] MLK-11354 sim: imx: relapce
 devm_request_and_ioremap with
 devm_ioremap_resource

commit  1eb26ac3bc625a16a0efb9365743ee6f3e00924c from
https://source.codeaurora.org/external/imx/linux-imx.git

The API devm_request_and_ioremap meets compile error
on branch imx_4.1.y. It is recommend to replace the api
with devm_ioremap_resource.

Signed-off-by: Gao Pan <b54642@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mxc/sim/imx_sim.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mxc/sim/imx_sim.c b/drivers/mxc/sim/imx_sim.c
index 75aa8bf..6a6c290 100755
--- a/drivers/mxc/sim/imx_sim.c
+++ b/drivers/mxc/sim/imx_sim.c
@@ -1770,7 +1770,7 @@ static int sim_probe(struct platform_device *pdev)
 		pr_err("Can't get the MEMORY\n");
 		return -ENOMEM;
 	}
-	sim->ioaddr = devm_request_and_ioremap(&pdev->dev, sim->res);
+	sim->ioaddr = devm_ioremap_resource(&pdev->dev, sim->res);
 	dev_dbg(&pdev->dev, "mapped base address: 0x%08x\n", (u32)sim->ioaddr);
 	if (IS_ERR(sim->ioaddr)) {
 		dev_err(&pdev->dev,
-- 
1.7.9.5

